Products/mcdutils/sessiondata.py

```diff
@@ -60,13 +60,15 @@ def addItemsToSession(self):
         """ Add key value pairs from 'items' textarea to the session.
         """
         request = self.REQUEST
-        items = request.form.get('items', ())
+        items = request.form.get('items', [])
```

Comment:
Why did you make this change?

Reply:
form:lines provides a list-type, right?

Reply:
That's correct, but makes no difference when all I need here is a default value for an empty iterable.

Reply:
Also `()` is not an empty tuple, but just braces that get evaluated to `None`. Empty tuple would need to be `tuple()`

Reply:
> Also `()` is not an empty tuple, but just braces that get evaluated to `None`. Empty tuple would need to be `tuple()`

Nein, das folgende funktionert in Python 2 und Python 3 korrekt:

```
>>> x = {}
>>> type(x.get('items', ()))
<class 'tuple'>
```

Reply:
Oh wow. TIL. So this is a special case in the python grammar that I completely forgot about. At least my memory was correct regarding this case:

```python
>>> type((None))
<class 'NoneType'>
```